sjb/config/common/test_cases/origin_release_with_ecosystem.yml

```diff
@@ -57,6 +60,12 @@ extensions:
       script: |-
         make build-images
         docker tag openshift/origin-web-console:latest "openshift/origin-web-console:$( cat ./ORIGIN_COMMIT )"
+    - type: "script"
```

Comment:
No, we should not be doing this.  Yo should be tagging in the latest from origin.